hack/make-rules/tools.mk

```diff
@@ -86,6 +86,13 @@ $(TOOLBIN)/misspell:
 	GOBIN=$(ABSTOOLBIN) go install github.com/client9/misspell/cmd/misspell@v0.3.4
 	$(call post-install-check)
 
+INSTALL_TOOLS += $(TOOLBIN)/goimports
+$(TOOLBIN)/goimports:
+	GOBIN=$(ABSTOOLBIN) go install golang.org/x/tools/cmd/goimports@v0.1.8
+	$(call post-install-check)
+	echo $(ABSTOOLBIN)/goimports -local fybrik.io/fybrik -w '$$'1 > $(ABSTOOLBIN)/fixgoimports.sh
+	chmod 755 $(ABSTOOLBIN)/fixgoimports.sh
+
```

Comment:
I personally never liked the fact that tools.ml installs `dlv`. Now `goimports` falls into the same category in my opinion. The original idea was to have tools installed just for the project so they won't compete with a global install of the user. But in my opinion that doesn't hold for tools that I already use in the project because my IDE does (like `dlv` and `goimports`). Just a personal opinion though.

Reply:
indeed, but I wanted to allow to configure/fix imports separately. 
Plan to remove it later.